integration-test/src/test/scala/io/phdata/clients/EmailClientImplIntegrationSpec.scala

```diff
@@ -15,7 +15,7 @@ class EmailClientImplIntegrationSpec extends FlatSpec {
       .send("test email",
             "<div>Hello from Archway.EmailClientImplIntegrationSpec</div>",
             "valhalla@phdata.io",
-            "tony@phdata.io")
+            "rkaland@phdata.io")
```

Comment:
It's better to use `itestConfig.approvers.notificationEmail.head` like we do in a test below instead of hardcoded value